gui/src/middleware.ts

```diff
@@ -1,25 +1,16 @@
-import { NextResponse, NextRequest } from 'next/server';
-import { parse } from 'cookie';
+import { NextRequest } from 'next/server';
 
 export function middleware(request: NextRequest) {
-  const cookies = parse(request.headers.get('cookie') || '');
-  const accessToken = cookies.accessToken;
-
-  if (request.nextUrl.pathname === '/') {
-    if (accessToken) {
-      const url = request.nextUrl.clone();
-      url.pathname = '/projects';
-      return NextResponse.redirect(url);
+  const token = request.cookies.get('token')?.value;
+  if (!token) {
```

Comment:
we can replace this nested if statement with two single if statements like

!token && request.nextUrl.pathname !== '/'
token && request.nextUrl.pathname === '/'

because you will either have token value or not, so there are only two possibilties

Reply:
This is more readable.